src/components/ha-climate-state.js

```diff
@@ -29,18 +29,24 @@ class HaClimateState extends LocalizeMixin(PolymerElement) {
           font-weight: bold;
           text-transform: capitalize;
         }
+
+        .unit {
+          display: inline-block;
```

Comment:
Why should we change the display property?

Reply:
Because otherwise the div causes a line break